README.md

```diff
@@ -4,10 +4,15 @@ Up makes it easier to start, build, and run Docker for local development.
 
 * **Much shorter commands.** For example, `up node test` instead of
   `docker-compose up run --rm web node test`
-* **Automatic image rebuilds.** No more pulling from git and wondering why your
-  project isn't working. Up will track necessary files and rebuild for you.
+* **Automatic image rebuilds.** When files your image depends on are changed 
+  (`package.json`, `Dockerfile`, etc.), Up will  rebuild the images before
+  running commands. No more pulling from git and wondering why your project 
+  isn't working (oops, forgot to run `docker-compose build`). 
```

Comment:
How is this? Added some examples to maybe clear it up a bit more

Reply:
nice! Yeah, that makes a lot more sense.

Reply:
Awesome! Thanks for the feedback!